test/tree.test.ts

```diff
@@ -125,15 +125,17 @@ test.each([
       "kiegroup/kie-uberfire-extensions",
       "kiegroup/droolsjbpm-knowledge",
       "kiegroup/drools",
-      "kiegroup/optaplanner",
```

Comment:
i found the removal of optaplanner on project dependencies definitions so it is expected now to be green.